app/controllers/api/permit_applications_controller.rb

```diff
@@ -340,12 +338,6 @@ def create_or_update_permit_block_status
 
   private
 
-  def determine_sandbox_id
-    return permit_application_params[:sandbox_id] if current_user.super_admin?
-
-    current_sandbox&.id
-  end
-
```

Comment:
should also update the front end to no longer pass in `sandbox_id` as a param

Reply:
fixed in a later branch